enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -414,17 +414,39 @@ The list of connected hubs should now reflect the disabled state.
 
 Uninstalling Federated Reporting from a superhub is not possible at this time.
 
-In order to remove Federated Reporting from a feeder remove the ```cftransport``` user
-and ```/opt/cfengine/federation``` directories. This will remove trust established
-with the superhub and will prevent superhub from performing any dump/import procedures.
+In order to remove Federated Reporting you must edit a configuration file on the
+feeder and allow cf-agent to make the needed changes either by waiting for an agent
```

Comment:
In master we tell people to edit the file?

Reply:
Adding docs for fr/hub-state

Reply:
@nickanderson updated, thanks!